src/bundle/Resources/public/js/scripts/admin.content.tree.js

```diff
@@ -94,5 +99,8 @@
     contentTreeContainer.addEventListener('transitioncancel', handleContainerTransitionStop, false);
     contentTreeContainer.addEventListener('transitionend', handleContainerTransitionStop, false);
 
-    window.addEventListener('resize', handleViewportChange, { capture: false, passive: true });
+    doc.body.addEventListener('ibexa-tb-rendered:ibexa-content-tree', removeContentTreeContainerWidth);
```

Comment:
Maybe `ibexa-tb-rendered:ibexa-content-tree` is correct, but it feels strange :D